AppCenterAuth/AppCenterAuth/Internals/Util/MSAuthConstants.h

```diff
@@ -25,3 +25,18 @@ static NSString *const kMSAuthConfigFilename = @"config.json";
  * The eTag key to store the eTag of current configuration.
  */
 static NSString *const kMSAuthETagKey = @"MSAuthETagKey";
+
+/**
+ * The AAD authority type string.
+ */
+static NSString *const kMSAuthorityTypeAAD = @"AAD";
```

Comment:
Move this to `MSAADAuthority.h`.

Reply:
I cannot define constants in header files unless I use extern keyword. Do you want me to use extern keyword for this?